swift-ci/sdks/static-linux/scripts/build.sh

```diff
@@ -268,6 +272,45 @@ else
     exit 1
 fi
 
+# -----------------------------------------------------------------------
+
+header "Patching BoringSSL"
+
+echo -n "Patching BoringSSL to disable -Werror... "
+sed -i.bak "s/-Werror //g" ${source_dir}/boringssl/CMakeLists.txt
+echo "done"
+
+# -----------------------------------------------------------------------
+
+header "Building clang for host"
+
+mkdir -p ${build_dir}/host/clang ${build_dir}/clang
+
+run cmake -G Ninja -S ${source_dir}/swift-project/llvm-project/llvm \
+    -B ${build_dir}/host/clang \
```

Comment:
Why does this work with the source dir being `llvm-project/llvm` instead of llvm-project/clang`?